src/components/ClassEnrollmentConfirm.tsx

```diff
@@ -0,0 +1,403 @@
+import React, { SetStateAction } from "react";
+import {
+    Box,
+    Button,
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Stack,
+    Flex,
+} from "@chakra-ui/react";
+import { BackButton } from "@components/BackButton";
+import { CloseButton } from "@components/CloseButton";
+import { Student, Parent } from "@prisma/client";
+import colourTheme from "@styles/colours";
+import parsePhoneNumber from "@utils/parsePhoneNumber";
+
+type ClassEnrollmentConfirmationProps = {
+    studentData: Student; // Not actually any but I don't want to type the entire student object, let me know if I have to do this or if it's done somewhere else that can be used here
+    parentData: any; // ^^^ Same idea
```

Comment:
Use the user type from `@prisma/client`
```suggestion
    parentData: User
```